KarhooUISDK/Screens/QuoteList/QuoteListFilters/Filters/ServiceAgreementsFilter.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ServiceAgreementsFilter.swift
+//  KarhooUISDK
+//
+//  Created by Aleksander Wedrychowski on 16/06/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import KarhooSDK
+
+extension QuoteListFilters {
+    enum ServiceAgreements: String, QuoteListFilter {
+        case freeCancelation = "free_cancellation"
+        case freeWatingTime = "free_waiting_time"
```

Comment:
Filters shall be aligned to final version within filter-section tickets, since @olivierkarhoo  is about to update requirements for Android as well. Let's say this is an initial version, based on current Android develop state.